x-pack/test/ingest_manager_api_integration/apis/epm/install_remove_assets.ts

```diff
@@ -0,0 +1,236 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import expect from '@kbn/expect';
+import { FtrProviderContext } from '../../../api_integration/ftr_provider_context';
+import { warnAndSkipTest } from '../../helpers';
+
+export default function ({ getService }: FtrProviderContext) {
+  const kibanaServer = getService('kibanaServer');
+  const supertest = getService('supertest');
+  const es = getService('es');
+  const dockerServers = getService('dockerServers');
+  const log = getService('log');
+  const pkgName = 'es_assets';
```

Comment:
As the package contains ES and kibana assets, would a name like `all_assets` make more sense?

Reply:
oops, forgot to update this!